### PR TITLE
add fireball cooldown message

### PR DESCRIPTION
**Pull request by ImYenil**
add fireball cooldown message

### CODE REVIEW

**Comment by tomkeuper** [comment]
this is only feasable if the delay is larger than a couple seconds otherwise it will spam you that you need to wait 0 seconds

**Comment by ImYenil** [comment]
> this is only feasable if the delay is larger than a couple seconds otherwise it will spam you that you need to wait 0 seconds

Changed this message to no longer appear if set to 0 seconds

File: bedwars-plugin/src/main/java/com/tomkeuper/bedwars/language/Bangla.java [hunk 1]
```diff
@@ -274,6 +274,7 @@ public Bangla() {
         yml.addDefault(Messages.PLAYER_DIE_REWARD_GOLD, "%bw_lang_prefix%&6+%bw_amount% %bw_meaning%");
         yml.addDefault(Messages.ARENA_MAX_BUILD_LIMIT_REACHED, "&cMax build height limit reached!");
         yml.addDefault(Messages.ARENA_MIN_BUILD_LIMIT_REACHED, "&cMin build height limit reached!");
+        yml.addDefault(Messages.ARENA_FIREBALL_COOLDOWN, "&cPlease wait %bw_cooldown%s to use that again!");
```
Comment (tomkeuper) [inline bedwars-plugin/src/main/java/com/tomkeuper/bedwars/language/Bangla.java:277]: Your string will look like: Please wait X to do that again. 

x what? I think you forgot to add the word seconds to all lang files

Comment (ImYenil) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/language/Bangla.java:277]: > Your string will look like: Please wait X to do that again.
> 
> x what? I think you forgot to add the word seconds to all lang files

image

**Comment by tomkeuper** [comment]
> > this is only feasable if the delay is larger than a couple seconds otherwise it will spam you that you need to wait 0 seconds
> 
> Changed this message to no longer appear if set to 0 seconds

I changed it to 1 second because I dont see the added benefit of it being send this fast when you just threw 1 fireball.

**Review by tomkeuper — APPROVED** [review]

**Review by porudev — APPROVED** [review]

**Pull request merged**